pythran/backend.py

```diff
@@ -573,7 +573,12 @@ def visit_Return(self, node):
                     Cxx.final_statement))
                 ])
         else:
-            stmt = ReturnStatement(self.visit(node.value))
+            value = self.visit(node.value)
+            if metadata.get(node, metadata.StaticReturn):
+                stmt = Block([Assign("static auto tmp_global", value),
+                              ReturnStatement("tmp_global")])
+            else:
+                stmt = ReturnStatement(value)
```

Comment:
What about adding a static qualifier to the type, and not modifying the back-end?